irondb/release-notes.md

```diff
@@ -5,6 +5,61 @@ sidebar_position: 12
 
 # Release Notes
 
+## Changes in 1.4.0
+
+2024-11-05
+
+**NOTE: This release deprecates legacy histograms. Histogram shards must be configured before
+upgrading to this release. If this is not done, nodes may not start up after the upgrade.**
+
+ * Fix use after free bug that could occasionally happen due to a race when fetching raw data.
+ * Fix potential memory leak on certain oil/activity data operations.
+ * Fix fetch bug where C style `calloc` allocations were being mixed with C++ styles `delete`s.
+ * Add new paramter to whisper config, `end_epoch_time`, that takes an epoch timestamp and directs
+   the code to not look in whisper files if the fetch time is after this time.
+ * Fix bug where histogram ingestion data was not being sent properly during rebalance operations.
+ * Fix bug where histogram rollup data was not being reconstituted during reconstitute operations.
+ * Add `get_engine` parameter to histogram data retrieval to allow pulling from either rollups
+   or ingestion data.
+ * Remove opening of new LMDB transaction when reading data for merging NNTBS blocks together.
+ * Remove support for mdbx as alternate NNTBS database type.
+ * Remove all support for legacy, non-sharded histograms.
+ * Fix bug where if a raw shard rollup was aborted after being scheduled but before actually starting,
+   multiple rollups could end up triggering at once.
+ * Fix rename bug where type was not getting set when failing to send NNTBS data.
+ * Add header to the `/rename` endpoint to `X-Snowth-Activity-Data-Mode`, which can accept either
+  `use_existing` or `create_new` as values.
+ * Treat `MDB_CORRUPTED`, `MDB_PAGE_FULL`, `MDB_TXN_FULL`, `MDB_BAD_TXN`, and `ENOMEM` as LMDB
+   corruption consistently when checking for errors.
+ * Allow switching between a fixed activity data merge and disabling the activity window update when
+   using the `/merge/nntbs` endpoint to send data to a node.
```

Comment:
I don't quite parse this, though I think I understand the gist of it. Could we try something like "Allow toggling activity updates on or off..."?

Reply:
Changed to:
```
 * When using the `/merge/nntbs` endpoint to send data to a node, allow either updating the receiving
   node's activity data using the incoming NNTBS data or leaving it as is and not updating it.
```